### PR TITLE
Fix/remove non null assertion

### PR DESCRIPTION
**Pull request by m4thieulavoie**
# Pull Request

## 📖 Description

This PR removes the non-null assertions in the anchor region storybook file.

### 🎫 Issues

Fixes https://github.com/microsoft/fast/issues/5340

### General

- [x] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [ ] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

**Review by scomea — APPROVED** [review]

File: packages/web-components/fast-components/src/anchored-region/anchored-region.stories.ts [hunk 1]
```diff
@@ -46,60 +45,59 @@ addons.getChannel().addListener(STORY_RENDERED, (name: string) => {
...
+            {
+                id: "toggle-inset-horizontal",
+                attribute: "horizontal-inset",
+                firstValue: "true",
+                secondValue: "false",
+            },
+        ].forEach(({ id, attribute, firstValue, secondValue }) => {
+            const toggleElement = document.getElementById(id);
+            if (toggleElement) {
+                toggleElement.addEventListener("click", (e: MouseEvent) => {
```
Comment (robarbms) [inline packages/web-components/fast-components/src/anchored-region/anchored-region.stories.ts:92]:
```suggestion
                toggleElement.addEventListener("click", () => {
```

File: packages/web-components/fast-components/src/anchored-region/anchored-region.stories.ts [hunk 2]
```diff
@@ -46,60 +45,59 @@ addons.getChannel().addListener(STORY_RENDERED, (name: string) => {
...
+                    if (togglesRegion?.getAttribute(attribute) === firstValue) {
+                        togglesRegion.setAttribute(attribute, secondValue);
+                    } else {
+                        togglesRegion.setAttribute(attribute, firstValue);
+                    }
```
Comment (robarbms) [inline packages/web-components/fast-components/src/anchored-region/anchored-region.stories.ts:97]: You don't need a check for togglesRegion, it's checked above. You can can setAttribute once here.
```suggestion
                    togglesRegion.setAttribute(attribute, togglesRegion.getAttribute(attribute) === firstValue ? secondValue : firstValue);
```

**Review by robarbms — APPROVED** [review]

**Review by chrisdholt — APPROVED** [review]

**Comment by chrisdholt** [comment]
@m4thieulavoie looks like prettier and lint may be failing - can you try and run locally?

**Comment by m4thieulavoie** [comment]
> @m4thieulavoie looks like prettier and lint may be failing - can you try and run locally?

Sure! Will get back at it either today or this weekend, sorry for the delay 👍

**Pull request merged**